templates/patterns/page_header/page_header.ui_patterns.yml

```diff
@@ -51,8 +51,8 @@ page_header:
       preview: "page-header-pattern-background-preview.jpeg"
     overlay:
       type: "text"
-      label: "Overlay"
-      description: "Optional overlay on top of background image (can be 'light', 'dark'). Only used on EC sites"
+      label: "Overlay - Deprecated and will be removed in 5.x stable version."
+      description: "Optional overlay on top of background image (can be 'light', 'dark'). Not used in ECL 5.x anymore."
```

Comment:
Should we do anything on EWPP side? The select box for selecting an overlay no longer works. Do we already have a follow up ticket to fix that (hide this select box or some other adjustments)?

Reply:
It is fine.